macros/customizeLaTeX.pl

```diff
@@ -0,0 +1,84 @@
+=head1 NAME
+
+customizeLaTeX.pl - Defines default LaTeX constructs for certain mathematical
+                    ideas.
+
+=head1 DESCRIPTION
+
+The functions are loaded by default.  Any/all can be overridden 
+in your course's PGcourse.pl
+=cut
+
+##### Set theory macros
+sub set_minus{ 
+	#return "\\setminus";
+	return '-';
+};
+
+#####  Logic macros
+sub negate { 
+	return "\\sim";
+	#return "\\lnot";
+};
+
+sub implies {
+	return "\\Rightarrow";
```

Comment:
`\implies` would be better than `\Rightarrow`. Here's a discussion: https://tex.stackexchange.com/questions/47063/rightarrow-vs-implies-and-does-not-imply-symbol